internal/validation/mock_trace_validation.go

```diff
@@ -30,10 +30,11 @@ import (
 )
 
 var (
-	ErrInvalidTimestamp   = status.Error(codes.InvalidArgument, "start time must be before end time")
-	ErrMalformedTimestamp = status.Error(codes.InvalidArgument, "unable to parse timestamp")
-	ErrDuplicateSpanName  = status.New(codes.AlreadyExists, "duplicate span name")
-	requiredFields        = []string{"Name", "SpanId", "DisplayName", "StartTime", "EndTime"}
+	ErrInvalidTimestamp    = status.Error(codes.InvalidArgument, "start time must be before end time")
```

Comment:
These statuses are going to be renamed and un-exported in another PR (https://github.com/googleinterns/cloud-operations-api-mock/pull/28), leaving this as they are in this PR to avoid merge conflicts